pathod/pathoc.py

```diff
@@ -73,7 +73,7 @@ def __str__(self):
             parts.append("\tPubkey: %s bit %s" % (pk.bits(), t))
             s = certutils.SSLCert(i)
             if s.altnames:
-                parts.append("\tSANs: %s" % " ".join(s.altnames))
+                parts.append("\tSANs: %s" % b" ".join(s.altnames))
```

Comment:
Yes, so this is basically used only for visualization and debugging purposes, we can convert/keep everything to `str` (i.e. utf8).